R/MsBackendMzR.R

```diff
@@ -36,10 +38,11 @@ setMethod("saveMsObject", signature(object = "MsBackendMzR",
               if (file.exists(fl))
                   warning("Overwriting already present 'backend_data.txt' file")
               writeLines(paste0("# ", class(object)[1L]), con = fl)
-              suppressWarnings(
-                  write.table(object@spectraData,
-                              file = fl, sep = "\t", quote = FALSE,
-                              append = TRUE, row.names = FALSE))
+              if (nrow(object@spectraData))
+                  suppressWarnings(
+                      write.table(object@spectraData,
+                                  file = fl, sep = "\t", quote = FALSE,
```

Comment:
is there any particular reason to use `quote = FALSE`? Maybe `quote = TRUE` might be *better* because then character strings are encoded by `"character string"` instead of `character string` so the import method would immediately recognize the content as character?

Reply:
mmm good question. I'm wondering if I didn't get some error at some point. but i can't remember and I don't see why. I'll check that !